src/Logger.cc

```diff
@@ -525,7 +525,7 @@ QString Logger::oldName( const QString & filename, int no )
     QString oldName = filename;
 
     if ( oldName.endsWith( ".log" ) )
-	oldName.remove( QRegExp( "\\.log$" ) );
+	oldName.truncate( 4 );
```

Comment:
Good idea in principle, but please let's avoid "magic numbers" like 4 in the code when it's easy to show the reader of the code what it's supposed to mean with some thing like `sizeof ( ".log" )`.